apps/vaporgui/VLineEdit.h

```diff
@@ -30,6 +30,7 @@ class VLineEdit : public VContainer {
     // Note: VLineEdit does NOT support an integer type at this point.
     // When isDouble == false, it represents a string. 
     void SetIsDouble( bool isDouble );
+    void SetReadOnly(bool b);
```

Comment:
Document all public methods

Reply:
Done but shouldn't the other methods of this class be documented then?

Reply:
yes

Reply:
I only added the method SetReadOnly, I did not write the rest of this class.